src/api/status/public/postsCount.js

```diff
@@ -0,0 +1,16 @@
+const totalPosts = document.getElementById('totalPosts');
+const fetchPostsCount = () =>
+  fetch('dev.api.telescope.cdot.systems').then((res) => res.headers.get('x-total-count'));
+
+const getPostsCount = async () => {
+  try {
+    const total = await fetchPostsCount();
+    totalPosts.innerText = total || 'N/A';
```

Comment:
This should use [null-coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) `total ?? "N/A"`, as 0 is a falsy value.